camel/societies/role_playing.py

```diff
@@ -95,55 +99,132 @@ def __init__(
         self.with_critic_in_the_loop = with_critic_in_the_loop
         self.model_type = model_type
         self.task_type = task_type
+        self.task_prompt = task_prompt
+
+        self.specified_task_prompt: Optional[TextPrompt] = None
+        self.init_specified_task_prompt(assistant_role_name, user_role_name,
+                                        task_specify_agent_kwargs,
+                                        extend_task_specify_meta_dict,
+                                        output_language)
+
+        self.planned_task_prompt: Optional[TextPrompt] = None
+        self.init_planned_task_prompt(task_planner_agent_kwargs,
+                                      output_language)
+
+        sys_msg_generator = SystemMessageGenerator(
+            task_type=self.task_type, **(sys_msg_generator_kwargs or {}))
+        (init_assistant_sys_msg, init_user_sys_msg,
+         sys_msg_meta_dicts) = self.get_sys_message_info(
+             assistant_role_name, user_role_name, sys_msg_generator,
+             extend_sys_msg_meta_dicts)
+
+        self.assistant_agent: Optional[ChatAgent] = None
```

Comment:
@zechengz  Remove Optional and initialization by None, leave only type annotation